round4/team-14.md

```diff
@@ -0,0 +1,105 @@
+
+## Selling Points
+
+1. Aesthetics
+2. Gammification
+3. Customizeable View
+4. Minimal upkeep
+5. Task Organization
+
+## User Stories
+
+1. As a student, I want to choose an egg to hatch so that I would be motivated to do work to hatch it.
+2. As a user, I would like to choose a new egg so that the novelty would motivate me in the middle of the school year.
+3. As a student, I would like a stopwatch feature so that I can work uninterrupted by timers.
+4. As someone who is concerned about my productivity over time, I want to sum the time I have spent working so that I can track how much time spent on a single task or project.
+5. As a student with multiple homework assignments, I want to track multiple tasks or assignments so that I don't neglect one task while I am currently focused on another.
+6. As someone who struggles with procrastination, I want an accountability feature that would make me spread out my work over multiple days so that I am not rushing to finish by the deadline.
+7. As someone with multiple things to do, I want a feature that will mark priority or urgency so that rewards and punishments scale with the importance of the task.
+8. As a student, I want the timer to work like a pomodoro so that I can factor in breaks and not burn out.
+9. As an animal-lover, I want food/treat mechanics for when I complete objectives so that I can use positive reinforcement for continuing to focus.
+10. As someone who needs flexibility, I want to be able to change the length of the timer for focus and breaks so that I can create long and short focus sessions.
+11. As a student who studies for extended periods of time, I want an option to take long breaks when I have completed a certain number of objectives
+12. As an animal enthusiast, I would like to level up or grow my pet for positive motivation for working.
+13. As someone who works late nights, I would like the option to toggle between light and dark mode so that I reduce my eyestrain.
+14. As an educator, I would like for the mechanics to be child-friendly and fun so that I can introduce time-management to my younger students.
+15. As a student, I would like to be notified when I can take a break so that I am not constantly checking the timer when it is done.
+16. As a busy student, I would like to schedule study sessions in advance for certain tasks, and get notifications for when that time comes.
+17. As a user, I would like switch between the sidebar and timer badge to minimize distractions while working.
+18. As a student who has trouble focusing while doing homework, I want to set a daily time goal for me to do homework.
+19. As a student who has an exam coming up, I want the extension to keep me on track the day before.
+
+## Notes
+
+* Will potentially use the Pokémon concept instead of Tamagachi to have more sprite options.  Will be careful of not breaking copyright laws, where [fair media usage is outlined here](https://press.pokemon.com/en/Assets-Use-Terms).
```

Comment:
@ashfujiyama Using these copyrighted characters is going to be a pain, because you will self-limit your success: The only way to use these assets in a way that complies with their terms, is to promise never to be commercial.

Instead, use an open-source clone, like the beautiful [Tuxemon](https://www.tuxemon.org/):
![banner-03](https://github.com/CIS-3500/ideation/assets/11770469/b1a30e37-b1a3-440f-91b4-634288fc047b)

You can see [a full list of their creatures here](https://wiki.tuxemon.org/Category:Monster). Note that the assets themselves are Creative Commons, but the codebase of the game is GPL (non-commercial), but you wouldn't be using the latter.

Reply:
that's really cool!  In the end, one of our members volunteered to design the sprites so we don't have to worry about using copyrighted characters, but I had no idea Tuxemon existed and I want to look into this more during my free time.